app/grants/sync/helpers.py

```diff
@@ -55,7 +55,8 @@ def record_contribution_activity(contribution):
                 comment= subscription.comments
             )
 
-        successful_contribution(grant, subscription, contribution)
+        # note: commenting out for optimistic UI
+        # successful_contribution(grant, subscription, contribution)
```

Comment:
where is this moved to?  seems like it's not re-added anywhere

Reply:
this is taken care  https://github.com/gitcoinco/web/pull/7797/files#diff-f630b64d112cbde98b1992af074c13f88ce29c0f6a81dffdc2fdfa967fb8a2c6R2701

changing it to true adds it

`contribution = subscription.create_contribution(tx_id, True)`